SU-HSDO/stanford_mrc
# MRCD8-173 program views

# READY FOR REVIEW

# Summary
- Front-end styles and templates for three views on program/series pages (events, news, visitors)

# Needed By (Date)
- End of sprint (1/17)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that styles are working and that templates are populating the /patterns page

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-173

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
